orgs-using-d.dd

```diff
@@ -237,6 +237,15 @@ $(DIVC orgs-using-d center,
             $(FA_HIRING www.tripaneer.com/news/work-with-us-software-developer-d-language-)
         )
     )
+    $(DORG Underground Rekordz, https://undergroundrekordz.com/, undergroundrekordz.png,
+        Music production company,
+        $(FA_QUOTE
+            D is the power behind our website and certain products/services
+        )
+        $(LINK_ROW
+            $(FA_TESTIMONIAL) $(HTTP https://cooldown.io/blogs/entry/2-using-artificial-intelligence-to-create-beautiful-music-part-1/, Testimonial)
```

Comment:
If you use the HTTP/HTTPS macro, this will already add that part to the URL (I know it's weird).
So you either need to remove `https://` or simply use `LINK2` instead.

Reply:
Done now :)